.github/workflows/lint.yml

```diff
@@ -35,6 +35,11 @@ jobs:
         go-version: '1.20'
       id: go
       if: ${{ !contains(matrix.os, 'ARMHF') }}
+    - id: govulncheck
+      uses: golang/govulncheck-action@7da72f730e37eeaad891fcff0a532d27ed737cd4
```

Comment:
This is newer than `@v1`. It is needed for:
- `repo-checkout: false` option
- avoid attempting to re-install Go